deploy/aks/monitoring/perf/README.md

```diff
@@ -0,0 +1,109 @@
+## Performance Test Dashboard Setup
+
+This document serves as a guideline for setting up a performance test dashboard using Azure managed Prometheus and Grafana. We will be creating an AKS (Azure Kubernetes Service) cluster and installing Prometheus Pushgateway and Azure Monitoring Agent (AMA) to scrape performance test metrics and push them to Azure managed Prometheus. Additionally, we will set up an ingress controller with authentication to act as a proxy for Prometheus Pushgateway. Finally, we will create a Grafana dashboard by importing a predefined JSON model.
+
+Below are step-by-step instructions to set up a performance test dashboard using Azure managed Prometheus and Grafana. Follow these steps to configure your performance monitoring environment.
+
+#### Step 1: Clone the test-infra repo
+
+```bash
+git clone https://github.com/dapr/test-infra.git
+cd deploy/aks/monitoring/perf
+```
+
+#### Step 2: Login to Azure
+
+```bash
+az login
+```
+
+### Step 3: Set resource group name, location and use the same namespace (dapr-perf-metrics)
+
+```bash
+DAPR_PERF_RG={reosuceGroup}
+DAPR_PERF_LOCATION={region}
```

Comment:
Nitpicking I would use `<something>` instead of '{something}' to document something to be changed.

First because it forces the user to replace that: `{something}` is a valid bash string that parses without an error whereas '<something is not>':

```
$ echo <a>
bash: syntax error near unexpected token `newline'
$ echo '{a}'
{a}
````

Second because, to me at least, `{something}` makes me believe that you just forgot a `$` before that, and that you instead meant to refer to a env. variable named `something` as in `${something}`

Also, you have a typo in `reosuceGroup`

```suggestion
DAPR_PERF_RG=<resource group to be used>
DAPR_PERF_LOCATION=<insert region>
SUBSCRIPTION_ID=<subscription UUID to be used>
```